games/formic/GameManager.js

```diff
@@ -182,12 +182,14 @@ define([
 			foodRatio,
 			maxFrame,
 			teams,
+			stepFrame
```

Comment:
As noted above, I think this variable should live with the UI and be sent as part of the event, not kept on the game itself.